cookbooks/cdo-ruby/recipes/2.0-remove.rb

```diff
@@ -0,0 +1,25 @@
+# Uninstall Ruby 2.0.
```

Comment:
Are we going to have to have a 2.2-remove.rb etc. each time a new version comes along?